index.html

```diff
@@ -28,17 +28,98 @@ <h2>I'am a Software Developer</h2>
     <div class="columns-wrapper">
         <div class="columns">
             <ul>
-                <li><a href="#"><img id="img3" src="Vector.png" alt="#"></a></li>
-                <li><a href="#"><img id="img4" src="Group.png" alt="#"></a></li>
-                <li><a href="#"><img id="img5" src="1Vector.png" alt="#"></a></li>
-                <li><a href="#"><img id="img6" src="2Vector.png" alt="#"></a></li>
+                <li><a href="#"><img id="img3" src="Images/Vector.png" alt="#"></a></li>
+                <li><a href="#"><img id="img4" src="Images/Group.png" alt="#"></a></li>
+                <li><a href="#"><img id="img5" src="Images/1Vector.png" alt="#"></a></li>
+                <li><a href="#"><img id="img6" src="Images/2Vector.png" alt="#"></a></li>
 
 
 
 </ul>
 </div>
 </div>
 </div>
+<section class="grid-container">
+    <div class="grid-item1">
+<ul>
+ <li class="head">My Recent Works</li>
+ <li class="border"></li>
+</ul>  
+ 
+<div id="img7"></div>
+<div>
+<h2 class="post">Multi-Post Stories</h2>
+<p class="support">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+</div>
+<ul class="tag">
+<li id="tag1">css</li>
+<li id="tag2">html</li>
+  <li id="tag3">bootstrap</li>
+  <li id="tag4">Ruby</li>
+</ul>
+  <button class="btn" type="button">See Project</button>
+</div>
+  <div>
+  <div class="img8"></div>
+  <ul> 
+    <li class="post2">Profesional Art Printing Data</li>
+<li class="support2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</li>
+</ul>
+<ul class="tagB">
+<li id="tag5">html</li>
+  <li id="tag6">bootstrap</li>
+  <li id="tag7">Ruby</li>
+  <li><button class="btn1" type="button">See Project</button></li>
+</ul>
+</div>
+<div>
+  <div class="img8"></div>
+  <ul> 
+    <li class="post2">Profesional Art Printing Data</li>
+<li class="support2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</li>
+</ul>
+<ul class="tagB">
+<li id="tag8">html</li>
+  <li id="tag9">bootstrap</li>
+  <li id="tag10">Ruby</li>
+  <li><button class="btn1" type="button">See Project</button></li>
+</ul>
+</div>
+</section>
+      <section class="tab">
+        <div>
+       <h2 class="about">About me</h2>
+       <p class="support3">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.<p>
+       <button class="btn3" type="button">Get My Resume</button>
```

Comment:
- [ ] Please be sure your buttons follow the button styling: Active, Hover and Pressed on your preferred template ([see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471))✌✌
![screenshot (9)](https://user-images.githubusercontent.com/44624138/144241947-d6a06d6c-99e6-4426-aed2-b8f39698fbd2.png)